test/common/testlib.py

```diff
@@ -466,9 +466,13 @@ def run(self, result=None):
                 self.currentResult = result
                 super(MachineCase, self).run(result)
             except RetryError, ex:
-                self.doCleanups()
-                sys.stderr.write("{0}\n".format(ex))
-                continue
+                if retry < 5:
```

Comment:
The intent was to make it higher than the actual theoretical upper limit. And previously we only ever retried once. However that's changing (and for good reason). So lets make this loop here be a very high limit in the thousands. Given the test timeout, unbounded.